Makefile

```diff
@@ -87,8 +89,14 @@ install-rbac: manifests
 uninstall-rbac: manifests
 	kustomize build config/rbac | kubectl delete --ignore-not-found=true -f -
 
-# Deploy both controller and cleanup_agent to the cluster
-deploy: deploy-controller deploy-cleanup-agent
+# Deploy both controller and cleanup_agent or controller only based on
+# the environment variable
+deploy:
```

Comment:
I am not familiar with Makefile. This first `deploy:` seems to be empty. Maybe we can delete it?

Reply:
You are right, this line should be deleted.

Reply:
I would also leave a space after "deploy:" in the lines that follow.